File: 프로그래머스/0/120833. 배열 자르기/배열 자르기.py
def solution(numbers, num1, num2):
    return numbers[num1:num2+1]  # 리스트 슬라이싱을 이용하여 부분 리스트 추출

# 테스트 예제 실행
print(solution([1, 2, 3, 4, 5], 1, 3))  # [2, 3, 4]
print(solution([1, 3, 5], 1, 2))        # [3, 5]

File: 프로그래머스/0/120833. 배열 자르기/배열 자르기.java
import java.util.Arrays;

public class Solution {
    public static int[] solution(int[] numbers, int num1, int num2) {
        return Arrays.copyOfRange(numbers, num1, num2 + 1);
    }

    public static void main(String[] args) {
        System.out.println(Arrays.toString(solution(new int[]{1, 2, 3, 4, 5}, 1, 3))); // [2, 3, 4]
        System.out.println(Arrays.toString(solution(new int[]{1, 3, 5}, 1, 2)));       // [3, 5]
    }
}